code/game/gamemodes/dynamic/dynamic_rulesets_midround.dm

```diff
@@ -252,7 +252,7 @@
 	antag_flag_override = ROLE_FAMILIES
 	protected_roles = list("Prisoner", "Head of Personnel")
 	restricted_roles = list("Cyborg", "AI", "Security Officer", "Warden", "Detective", "Head of Security", "Captain", "Research Director")
-	required_candidates = 9
+	required_candidates = 0
```

Comment:
Didn't you say it requires 3 candidates now?

Reply:
I FORGOT TO PUSH